ema_workbench/analysis/parcoords.py

```diff
@@ -143,6 +143,27 @@ class ParallelAxes:
     rot : float, optional
           rotation of axis labels
 
+
+    Attributes
+    ----------
+    limits : DataFrame
+    recoding : dict
+               non numeric columns are converting to integer variables
+    flipped_axes : set
+                   set of Axes that are to be shown flipped
+    axis_labels : list of str
+    fontsize : int
+    normalizer : MinMaxScaler instance
```

Comment:
Add a description of `normalizer`

Reply:
I made this an implementation detail and removed it from the documentation. It is not part of the public API so you don't want to mess with the normalizer.